Show average star rating

bobsilverberg opened the pull request:
Fixes #5436 

Before:

![screenshot 2018-07-25 17 27 27](https://user-images.githubusercontent.com/142755/43228614-0526bfce-9030-11e8-9cbe-64a93351c78c.png)

After: 

![screenshot 2018-07-25 17 26 46](https://user-images.githubusercontent.com/142755/43228587-edb9c5ac-902f-11e8-98c9-e9048b1e1923.png)

rebmullin commented:
for some reason I am having issue with my image tool. here are the links:

https://www.evernote.com/shard/s300/sh/bee1d83b-1238-45bc-8d60-e059c8dbf473/5e179a888104c92b

https://www.evernote.com/l/ASw1Z3tUTHJJ07D5PB1RwTnIlBsJ-zzjFrs

rebmullin commented:
@bobsilverberg, I also see some flow check issues. So you should check that too :)

bobsilverberg commented:
Thanks for the reviews @rebmullin and @willdurand. I've addressed the concerns, but I also noticed, when looking at the issue, that there's an inconsistency in the requirements, so I've asked a couple of questions there. I'm going to mark this as blocked pending responses from @pwalm.

bobsilverberg commented:
Ok, this has been updated according to the comment at https://github.com/mozilla/addons-frontend/issues/5436#issuecomment-408123161.

willdurand commented on tests/unit/amo/components/TestAddonMeta.js:
```diff
@@ -114,6 +118,42 @@ describe(__filename, () => {
+      const average = 2.34;
+      const root = renderRatings({ average });
+
+      const averageMeta = root.find(MetadataCard).prop('metadata')[2];
```
Getting the data with an index looks a bit fragile, maybe we could create a helper function to `getAverageMeta({ root })`?
